main/wpa_supplicant/APKBUILD

```diff
@@ -1,37 +1,38 @@
 # Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=wpa_supplicant
-pkgver=2.6
-pkgrel=15
+pkgver=2.7
+pkgrel=0
 pkgdesc="A utility providing key negotiation for WPA wireless networks"
 url="https://w1.fi/wpa_supplicant/"
 arch="all"
 license="BSD"
 subpackages="$pkgname-doc $pkgname-openrc"
 makedepends="linux-headers openssl-dev dbus-dev libnl3-dev pcsc-lite-dev"
 source="https://w1.fi/releases/$pkgname-$pkgver.tar.gz
-	rebased-v2.6-0001-hostapd-Avoid-key-reinstallation-in-FT-handshake.patch
-	rebased-v2.6-0002-Prevent-reinstallation-of-an-already-in-use-group-ke.patch
-	rebased-v2.6-0003-Extend-protection-of-GTK-IGTK-reinstallation-of-WNM-.patch
-	rebased-v2.6-0004-Prevent-installation-of-an-all-zero-TK.patch
-	rebased-v2.6-0005-Fix-PTK-rekeying-to-generate-a-new-ANonce.patch
-	rebased-v2.6-0006-TDLS-Reject-TPK-TK-reconfiguration.patch
-	rebased-v2.6-0007-WNM-Ignore-WNM-Sleep-Mode-Response-without-pending-r.patch
-	rebased-v2.6-0008-FT-Do-not-allow-multiple-Reassociation-Response-fram.patch
-	rebased-v2.6-0001-WPA-Ignore-unauthenticated-encrypted-EAPOL-Key-data.patch
 
 	wpa_supplicant.initd
 	wpa_supplicant.confd
 	wpa_cli.confd
 	wpa_cli.initd
 
-	libressl.patch
 	eloop.patch
 
 	config
 	wpa_cli.sh"
 
 # secfixes:
+#   2.7-r0:
+#     - CVE-2017-13077
+#     - CVE-2017-13078
+#     - CVE-2017-13079
+#     - CVE-2017-13080
+#     - CVE-2017-13081
+#     - CVE-2017-13082
+#     - CVE-2017-13086
+#     - CVE-2017-13087
+#     - CVE-2017-13088
+#     - CVE-2018-14526
```

Comment:
looks like we fixed CVE-2018-14526 in 2.6-r14 (see line below)

Reply:
Removed, also fixed in commit message